### PR TITLE
Update master php testing versions

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -34,9 +34,8 @@ pipeline:
     secrets: [ github_token ]
     when:
       matrix:
-        include:
-        - TESTS: syntax-php7.0
         - TESTS: signed-off-check
```
Comment (MorrisJobke) [inline .drone.yml]: Could it be that the `- ` needs to be removed?

```
fatal: destination path 'server' already exists and is not an empty director
```

Comment (MorrisJobke) [reply .drone.yml]: Ah no - this is in the syntax check.

**Comment by nickvergessen** [comment]
@blizzz this is somehow broken, please have a look

**Comment by blizzz** [comment]
> \+ bash ./before_install.sh $APP_NAME $CORE_BRANCH $DB
Work directory: /drone/src/github.com/nextcloud/workflow_script
Database: sqlite
Cloning into 'server'...
Submodule '3rdparty' (https://github.com/nextcloud/3rdparty.git) registered for path '3rdparty'
Cloning into '3rdparty'...
fatal: reference is not a tree: 323375f4da08940c16a48e174ac1271aebcb1202
Unable to checkout '323375f4da08940c16a48e174ac1271aebcb1202' in submodule path '3rdparty'

that's fishy, but nothing really caused by this app's code

**Pull request merged**